subsys/mgmt/osdp/src/osdp_phy.c

```diff
@@ -354,16 +354,13 @@ int osdp_phy_check_packet(struct osdp_pd *pd, uint8_t *buf, int len,
 			pd->seq_number = -1;
 			sc_deactivate(pd);
 		}
+#ifdef CONFIG_OSDP_MODE_PD
```

Comment:
Let's not have these `#ifdefs` in code please, It took a lot of effort to get rid of most of them :). In this case, you can remove `pd->resend` and introduce a new pd flag `PD_FLAG_REPLY_RESEND` (as we do in other places) and use that instead.